Source/CarthageKit/Availability.swift

```diff
@@ -45,36 +46,31 @@ public func duplicateProjectsInCartfiles(_ cartfile1: Cartfile, _ cartfile2: Car
 public enum ProjectIdentifier {}
 
 // MARK: - Git.swift
-
```

Comment:
I've changed the custom rule to enforce a single empty line before and after any MARK: comment.